pkg/crd/manifest.go

```diff
@@ -14,28 +14,63 @@
  * limitations under the License.
  */
 
-package core
+package crd
 
 import (
 	"errors"
 	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/jinzhu/copier"
+	"github.com/projectriff/riff/pkg/fileutils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"net/url"
 	"path/filepath"
+	"strings"
+)
 
-	"github.com/projectriff/riff/pkg/fileutils"
+type ResourceChecks struct {
+	Kind     string               `json:"kind,omitempty"`
+	Selector metav1.LabelSelector `json:"selector,omitempty"`
+	JsonPath string               `json:"jsonpath,omitempty"`
+	Pattern  string               `json:"pattern,omitempty"`
+}
 
-	"github.com/ghodss/yaml"
-)
+type Resource struct {
+	Path      string           `json:"path,omitempty"`
+	Content   string           `json:"content,omitempty"`
+	Name      string           `json:"name,omitempty"`
+	Namespace string           `json:"namespace,omitempty"`
```

Comment:
How are Name and Namespace used? `Resource` here represents a config file that may contain multiple k8s resources. Each of those resources will have a name and namespace (if applicable)